pages/index.html

```diff
@@ -1,12 +1,93 @@
+  <link rel="stylesheet" href="/style/style.css">
+  <link rel="stylesheet" href="/style/index.css">
+  <title>Jeannie</title>
 </head>
+
 <body>
-  
+  <header class="main-header">
+    <nav class="main-nav">
+      <ul>
+        <li><a href="/pages/index.html">Home</a></li>
+        <li><a href="/pages/about.html">About</a></li>
+        <li><a href="/pages/portfolio.html">Portfolio</a></li>
+      </ul>
+    </nav>
+    <hr>
+    <h1 class="name">Jeannie Zheng</h1>
+  </header>
+  <br> <br>
+  <section class="about-me">
+    <h2>About Me</h2>
+    <div><a href="/pages/about.html" target=blank>
+        <img src="/images/sza.png"></a>
+    </div>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna
+      aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis
+      aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+      occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor
+      sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna
+      aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis
+      aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+      occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+    <br>
+
+
+  </section>
+
+  <section class="portfolio">
+    <h2>Portfolio</h2>
+    <div><a href="/pages/portfolio.html" target=blank>
+        <img src="/images/code.png" alt=""></a>
+    </div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna
+      aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis
+      aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+      occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor
+      sit
+      amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+      minim
+      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+      in
+      reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non
+      proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+
+  </section>
+
+  <footer class="main-footer">
+    <div class="container main-footer-container">
```

Comment:
I would suggest "combining" these two elements together. The `footer` already acts like a divider, like `section` or `div`.